dev/build.image/build.gradle

```diff
@@ -668,183 +726,10 @@ publishing {
             version project.version
             artifact zipOpenLibertyDev
         }
-        if (isAutomatedBuild && !isIFIXBuild) {
```

Comment:
Why removing all the Maven stuff?

Reply:
I checked with Chuck, it was never used. The only maven artifact used is the general `openliberty` one ... and that is only used as a pre-req to the WS-CD build